app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Api::V1::ReservationsController < ApplicationController
+  before_action :find_reservation, only: %i[show update destroy]
+  # GET /reservations by user
+  def index
+    @reservations = Reservation.all
```

Comment:
Hi, @dgcuenca:
Great work so far!!!

I think that you only need to return the reservations of the selected user.

The rationale behind this is that in our routes we have `reservations` nested inside `users`, so you have for the index action the following route:

`/api/v1/users/:user_id/reservations(.:format)`

So, please kindly update this or let me know if you have any concerns about it.

Reply:
Hello @paulsaenzsucre 

Changes where implemented, thanks for the feedback.